Error running test:e2e:ci

```
$ npm run test:e2e:ci

> js-ipfs-www@0.0.0 test:e2e:ci /Users/alan/Code/protocol-labs/js.ipfs.io
> start-server-and-test serve http://localhost:9000 cy:run

starting server using command "npm run serve"
and when url "http://localhost:9000" is responding
running tests using command "cy:run"

> js-ipfs-www@0.0.0 serve /Users/alan/Code/protocol-labs/js.ipfs.io
> gatsby serve

info gatsby serve running at: http://localhost:9000/

> js-ipfs-www@0.0.0 cy:run /Users/alan/Code/protocol-labs/js.ipfs.io
> cypress run

Cypress could not verify that the server set as your 'baseUrl' is running:

  > http://localhost:8000

Your tests likely make requests to this 'baseUrl' and these tests will fail if you don't boot your server.

Please start this server and then run Cypress again.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! js-ipfs-www@0.0.0 cy:run: `cypress run`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the js-ipfs-www@0.0.0 cy:run script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/alan/.npm/_logs/2019-01-28T10_47_06_378Z-debug.log
{ Error: Command failed: npm run cy:run
    at makeError (/Users/alan/Code/protocol-labs/js.ipfs.io/node_modules/start-server-and-test/node_modules/execa/index.js:174:9)
    at Promise.all.then.arr (/Users/alan/Code/protocol-labs/js.ipfs.io/node_modules/start-server-and-test/node_modules/execa/index.js:278:16)
    at process._tickCallback (internal/process/next_tick.js:68:7)
  code: 1,
  stdout: null,
  stderr: null,
  failed: true,
  signal: null,
  cmd: 'npm run cy:run',
  timedOut: false,
  killed: false }
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! js-ipfs-www@0.0.0 test:e2e:ci: `start-server-and-test serve http://localhost:9000 cy:run`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the js-ipfs-www@0.0.0 test:e2e:ci script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/alan/.npm/_logs/2019-01-28T10_47_06_484Z-debug.log
```

--- a/package.json
+++ b/package.json
@@ -20,7 +20,6 @@
     "cy:open": "cypress open",
     "cy:run": "cypress run",
     "test:e2e": "start-server-and-test develop http://localhost:8000 cy:open",
-    "test:e2e:ci": "start-server-and-test serve http://localhost:9000 cy:run",
     "test:e2e:hook": "start-server-and-test develop http://localhost:8000 cy:run"
   },
   "dependencies": {

